### PR TITLE
Suggest installing CLIMADA via conda in installation instructions

### LINKED ISSUE #673: Make `conda install climada` the default installation instruction
Climada is now available via conda-forge: https://anaconda.org/conda-forge/climada

This means that the tedious dependency installation before running `pip install climada` can now be dropped in favor of the much simpler `conda install -c conda-forge climada`. To get to the same result as with the current [simple installation instructions](https://climada-python.readthedocs.io/en/stable/guide/install.html#simple-instructions), one now has to execute a single command:

```
conda create -n climada_env -c conda-forge climada
```

This will install Climada and all its dependencies into a new `climada_env` environment.

Proposals:
* Add `conda install -c conda-forge climada` as installation instruction in the README
* Rewrite the [Simple Instructions](https://climada-python.readthedocs.io/en/stable/guide/install.html#simple-instructions) to simply use  `conda create -n climada_env -c conda-forge climada`

### PR DESCRIPTION
**Pull request by peanutfun**
Changes proposed in this PR:
- Instead of building the `climada_env` and installing CLIMADA via `pip`, the "Simple Instructions" in the installation instructions now advise users to simply install CLIMADA via `conda`. This only requires a single command.
- Advertise installation with `conda` also in the `README.md`.

This PR fixes #673

### PR Author Checklist

- [x] Read the [Contribution Guide][contrib]
- [x] Correct target branch selected (if unsure, select `develop`)
- [x] Descriptive pull request title added
- [x] Source branch up-to-date with target branch
- [x] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
- ~~[ ] [Tests][testing] updated~~
- [x] [Tests][testing] passing
- [x] No new [linter issues][linter]
- [x] [Changelog](https://github.com/CLIMADA-project/climada_python/blob/main/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/stable/guide/Guide_Continuous_Integration_and_Testing.html#3.C.--Static-Code-Analysis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,13 @@ This is the Python (3.8+) version of CLIMADA - please see https://github.com/dav
 
 ## Getting started
 
-CLIMADA runs on Windows, macOS and Linux. It can be installed from sources or - in case of climada_python - directly with pip. See the [installation guide](https://climada-python.readthedocs.io/en/latest/guide/install.html) for instructions.
+CLIMADA runs on Windows, macOS and Linux.
+The released versions can be installed directly through Anaconda:
```
Comment (chahank) [inline README.md:23]:
```suggestion
The released versions of the CLIMADA core can be installed directly through Anaconda:
```

File: doc/guide/install.rst [hunk 2]
```diff
@@ -25,9 +25,6 @@ Prerequisites
   All dependencies will be downloaded from the internet.
   Do **not** use a metered, mobile connection!
 * Install `Anaconda`_, following the `installation instructions <https://docs.anaconda.com/anaconda/install/>`_ for your OS.
-* Create a **workspace directory**.
-  To make sure that your user can manipulate it without special privileges, use a subdirectory of your user/home directory.
-  Do **not** use a directory that is synchronized by cloud storage systems like OneDrive, iCloud or Polybox!
 * **Linux** users need to make sure they have ``git`` and ``curl`` installed.
   Ubuntu and Debian users may use APT:
```
Comment (chahank) [inline doc/guide/install.rst:29]: This is not needed anymore with the conda install. Should we move it to the section for the advanced installation?

**Comment by chahank** [comment]
Thanks for the conda install @peanutfun !

**Comment by bguillod** [comment]
Out of curiosity: Why do we now want to recommend to users to install CLIMADA via conda, which is (in my experience at least) much more cumbersome than via pip?

**Comment by chahank** [comment]
CLIMADA can only really be installed using conda at the moment (the climada_env needs to be made with conda). So I am not sure to understand what you mean?

File: doc/guide/install.rst (hunk 2 above)
Comment (peanutfun) [reply doc/guide/install.rst:29]: Indeed, will fix this!

Comment (peanutfun) [reply doc/guide/install.rst:29]: @chahank there is no need to download `env_climada.yml` as a single file anymore. I think we can ditch ``curl`` then, right?

File: doc/guide/install.rst [hunk 3]
```diff
@@ -120,6 +85,21 @@ Advanced Instructions
 
 For advanced Python users or developers of CLIMADA, we recommed cloning the CLIMADA repository and installing the package from source.
 
+#. If you are using a **Linux** OS, make sure they have ``git`` installed
```
Comment (peanutfun) [inline doc/guide/install.rst:88]:
```suggestion
#. If you are using a **Linux** OS, make sure you have ``git`` installed
```

File: doc/guide/install.rst (hunk 2 above)
Comment (chahank) [reply doc/guide/install.rst]: Good point! Agreed!

Comment (peanutfun) [reply doc/guide/install.rst]: Done

File: README.md [hunk 4]
```diff
@@ -19,7 +19,13 @@ This is the Python (3.8+) version of CLIMADA - please see https://github.com/dav
 
 ## Getting started
 
-CLIMADA runs on Windows, macOS and Linux. It can be installed from sources or - in case of climada_python - directly with pip. See the [installation guide](https://climada-python.readthedocs.io/en/latest/guide/install.html) for instructions.
+CLIMADA runs on Windows, macOS and Linux.
+The released versions of the CLIMADA core can be installed directly through Anaconda:
+```shell
+conda install -c conda-forge climada
+```
+It is **highly recommend** to install CLIMADA into a **separate** Anaconda environment.
```
Comment (peanutfun) [inline README.md:27]:
```suggestion
It is **highly recommended** to install CLIMADA into a **separate** Anaconda environment.
```

**Comment by peanutfun** [comment]
@chahank Did you finish your review?

**Comment by chahank** [comment]
Yes, thanks!

**Pull request merged**